<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>39</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="macSpoofPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Interface:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="interfaceEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>MAC Address:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addressEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="spoofClickMe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Spoof!</string>
      </property>
     </widget>
     <widget class="QLabel" name="mac_update">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>551</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mac_current">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>591</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mac_page_title">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MAC Changer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="arpSpooferPage">
     <widget class="QLineEdit" name="targetIP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>311</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="gatewayIP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>311</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="arpSpoofButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Spoof!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Target IP address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>191</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Gateway IP address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="arpSpoofUpdate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Restore ARP</string>
      </property>
     </widget>
     <widget class="QLabel" name="arp_spoof_title">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ARP Spoofer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="networkScannerPage">
     <widget class="QPushButton" name="scanButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scanIPEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>301</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter the target IP Address:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetDevices">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>641</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Network Scanner</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="packetSnifferPage">
     <widget class="QPushButton" name="startPacketSniff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Packet Sniff!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopPacketSniff">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Packet Sniff!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>581</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Run the ARP spoofer along side to view incoming packets from the targeted system!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="interfaceSniff">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Interface:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="result_Browser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>641</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sniff_error_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>148</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="packet_sniff_title">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Packet Sniffer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pcapPage">
     <widget class="QLineEdit" name="pcapInterfaceSniff">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>321</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Packet Sniffer (PCAP file creator)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Interface:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>FIleName:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pcapFilename">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>321</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pcapUpdateLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>651</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="sniffPCAPFile">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Sniff!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopPCAPSniff">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Sniff!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>201</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of packets to capture:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="packet_count">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="macButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Mac Changer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arpButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP Spoofer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="networkButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Network Scanner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="packetButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Packet Sniffer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pcapButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PCAP file creator</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
